psi4/src/psi4/cc/cceom/diag.cc

```diff
@@ -930,10 +930,12 @@ void diag() {
 
         outfile->Printf("\nProcedure converged for %d root(s).\n", num_converged);
         if (num_converged == eom_params.cs_per_irrep[C_irr]) {
+            // all good
         } else if (iter == eom_params.max_iter) {
             outfile->Printf("\nMaximum number of iterations exceeded, ");
             outfile->Printf("so not all roots converged!\n\n");
         } else if ((params.wfn == "EOM_CC3") && (num_converged == 1)) {
+            // all good
```

Comment:
What are these comments for?

Reply:
lgtm doesn't like blank `if {}` blocks. not sure what the recc. systematic fix should be.